org.eclipse.xtext.full.releng/pom.xml

```diff
@@ -0,0 +1,162 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.xtext</groupId>
+		<artifactId>xtext-parent</artifactId>
+		<version>2.31.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.eclipse.xtext.full.releng</artifactId>
+	<packaging>pom</packaging>
+
+	<description>Builds everything</description>
+
+	<modules>
+		<module>..</module>
+		<module>../org.eclipse.xtext.nodeploy.parent</module>
+		<module>../org.eclipse.xtext.dev-bom</module>
+		<module>../org.eclipse.xtext.xbase.lib</module>
+		<module>../org.eclipse.xtext.xbase.lib.gwt</module>
+		<module>../org.eclipse.xtend.lib.macro</module>
+		<module>../org.eclipse.xtend.lib</module>
+		<module>../org.eclipse.xtend.lib.gwt</module>
+
+		<module>../org.eclipse.xtext.xbase.lib.feature</module>
+		<module>../org.eclipse.xtext.xbase.lib.tests</module>
+		<module>../org.eclipse.xtend.lib.feature</module>
+		<module>../org.eclipse.xtend.lib.tests</module>
+		<module>../org.eclipse.xtend.lib.gwt.test</module>
+
+		<!-- xtext-core -->
+		<module>../org.eclipse.xtext</module>
+		<module>../org.eclipse.xtext.util</module>
+		<module>../org.eclipse.xtext.ide</module>
+		<module>../org.eclipse.xtext.testing</module>
+		<module>../org.eclipse.xtext.xtext.ide</module>
+		<module>../org.eclipse.xtext.xtext.generator</module>
+		<module>../org.eclipse.xtext.xtext.wizard</module>
+		<!-- This is required by the xtext-maven-plugin IT -->
+		<module>../org.eclipse.xtext.testlanguages</module>
+
+		<!-- xtext-core -->
+		<module>../org.eclipse.xtext.tests</module>
+		<module>../org.eclipse.xtext.ide.tests</module>
+		<module>../org.eclipse.xtext.testlanguages.ide</module>
+		<module>../org.eclipse.xtext.xtext.bootstrap</module>
+
+		<!-- xtext-extras -->
+		<module>../org.eclipse.xtext.smap</module>
+		<module>../org.eclipse.xtext.common.types</module>
+		<module>../org.eclipse.xtext.ecore</module>
+		<module>../org.eclipse.xtext.xbase</module>
+		<module>../org.eclipse.xtext.xbase.ide</module>
+		<module>../org.eclipse.xtext.xbase.testing</module>
+		<module>../org.eclipse.xtext.builder.standalone</module>
+		<module>../org.eclipse.xtext.generator</module>
+		<module>../org.eclipse.xtext.java</module>
+		<!-- This is required by the xtext-maven-plugin IT -->
+		<module>../org.eclipse.xtext.purexbase</module>
```

Comment:
we also publish this to maven central meanwhile

Reply:
Yes, we also publish that.
With the monorepo I think it's not needed anymore to publish that, but we can leave it like that for the moment.
Or did you mean anything else?

Reply:
am not sure about that. but i assume it does not harm to keep publishing it

Reply:
Let's say that if someone wants to experiment with the xtext-maven-plugin with an existing example Xbase-DSL they can find purexbase on Central :)